pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -254,12 +253,17 @@ func (c *Cluster) makeCmdProxySidecarContainer(mgrConfig *mgrConfig) v1.Containe
 	return container
 }
 
-func getDefaultMgrLivenessProbe() *v1.Probe {
+func getDefaultMgrLivenessProbe(daemonId string) *v1.Probe {
 	return &v1.Probe{
 		Handler: v1.Handler{
-			HTTPGet: &v1.HTTPGetAction{
-				Path: "/",
-				Port: intstr.FromInt(int(DefaultMetricsPort)),
+			Exec: &v1.ExecAction{
+				Command: []string{
+					"env",
+					"-i",
+					"sh",
+					"-c",
+					fmt.Sprintf("ceph daemonperf mgr.%s 0 1", daemonId),
```

Comment:
A comment explaining what the `0` `1` args mean here would be useful.

Reply:
What part of the mgr does this command exercise? The module that collects metrics? And this uses the admin socket, right?

Reply:
If it's calling the socket we need to use `GenerateLivenessProbeExecDaemon` instead.

Reply:
@travisn : The command is used to watch and retrieve performance counters from a Ceph daemon. And yes, internally uses the admin socket to communicate with the daemon.

Reply:
@leseb: I think we do not need to use `GenerateLivenessProbeExecDaemon`  as we do in the controller, because the `daemonperf` command do not need the admin socket as parameter, and the daemon id is very easy to obtain.

Reply:
I guess it'll be better to have a common code when possible. We don't need to do things differently here, especially if `daemonperf` is calling the socket. So I still think adapting `GenerateLivenessProbeExecDaemon` is important for consistancy.